mozjs-sys/Cargo.toml

```diff
@@ -2,7 +2,7 @@
 name = "mozjs_sys"
 description = "System crate for the Mozilla SpiderMonkey JavaScript engine."
 repository.workspace = true
-version = "0.128.3-0"
+version = "0.128.4-0"
```

Comment:
```suggestion
version = "0.128.3-1"
```

I believe the version before the dash is meant to correspond with the upstream LTS version.

Reply:
That's right.